controllers/grants.go

```diff
@@ -30,15 +33,15 @@ func GrantsRouter(service *grant.Service) chi.Router {
 		if err != nil {
 			panic("THROTTLE_GRANT_REQUESTS was provided but not a valid number")
 		}
-		r.Method("POST", "/", chiware.Throttle(int(throttle))(middleware.InstrumentHandler("RedeemGrants", RedeemGrants(service))))
+		r.Method("POST", "/", chiware.Throttle(int(throttle))(RedeemGrants(service)))
 	} else {
-		r.Method("POST", "/", middleware.InstrumentHandler("RedeemGrants", RedeemGrants(service)))
+		r.Method("POST", "/", RedeemGrants(service))
 	}
 	// Hacky compatibility layer between for legacy grants and new datastore
-	r.Method("GET", "/active", middleware.InstrumentHandler("GetActive", GetActive(service)))
-	r.Method("POST", "/drain", middleware.InstrumentHandler("DrainGrants", DrainGrants(service)))
-	r.Method("POST", "/claim", middleware.InstrumentHandler("ClaimGrant", Claim(service)))
```

Comment:
let's match the method names to the names that were previously passed just so we have continuity of data

Reply:
corrected here: https://github.com/brave-intl/bat-go/pull/296/commits/b2d705881830c8658e40d53260a8f5e9f608308d